### ISSUE TITLE
Running Multiple files 

### ISSUE THREAD

**Issue by ShivaKumar-Jadipalli** [MicrosoftDocs/visualstudio-docs#9731]
Hi Sorry for asking this but I can't understand how to run multiple files in visual studio ide like I created a solution file and then created two files named testing_1.py and testing_2.py. When I run testing_1.py it works fine but when I run testing_2.py using shortcut ctrl + f5 it is still running testing_1.py, how to solve this and if possible please suggest me a video/text tutorial for learning on "how to get started with visual studio " other than documentation cause I am a beginner and it is kind of hard to follow documentations

**Comment by WilliamAntonRohm** [comment]
@ShivaKumar-Jadipalli -- Shiva, thank you for your feedback.  Please try starting a second instance of Visual Studio and running testing_2.py there.

Here are some resources; unfortunately they are mostly text:
- Ask a question or find an answer on the [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/tags/176/vs) page for Visual Studio.
- Check the [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio). 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

I am closing this issue now.  You are welcome to @ mention me for any followup.  We hope to hear from you again.
